e2e/rbd.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 	"time"
 
+	. "github.com/ceph/ceph-csi/internal/util"
```

Comment:
I don't see any problem importing without `.` in my dev setup. Is this something vim doing automatically?

Reply:
If I don't put `.` vim is removing the whole import line, crazy :-)

Reply:
in vscode it's fine. it's not removing it and there is no complaint in linter about this also.